gemini/use-cases/retrieval-augmented-generation/rag_evaluation/ragas_with_gemini.ipynb

```diff
@@ -0,0 +1,816 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "executionInfo": {
+     "elapsed": 3,
+     "status": "ok",
+     "timestamp": 1708026667086,
+     "user": {
+      "displayName": "",
+      "userId": ""
+     },
+     "user_tz": 300
+    },
+    "id": "ijGzTHJJUCPY",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Copyright 2024 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VEqbX8OhE8y9"
+   },
+   "source": [
+    "# Getting Started with RAGAS & Vertex AI Gemini API\n",
+    "\n",
+    "<table align=\"left\">\n",
+    "  <td style=\"text-align: center\">\n",
+    "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/generative-ai/blob/main/language/rag_evaluation/ragas_with_gemini.ipynb\">\n",
```

Comment:
Make sure to change these links to match the correct location/notebook name

Reply:
Done